generators/app/templates/src/app/components/RadioGroup/components/RadioOption.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function RadioOption({
+  className,
+  inputClassName,
+  labelClassName,
+  value,
+  label,
+  name,
+  checked,
+  disabled,
+  onChange,
+  children
+}) {
+  return (
+    <div className={className}>
+      <input
+        type="radio"
+        id={label}
+        className={inputClassName}
+        value={value}
+        name={name}
+        checked={checked}
+        disabled={disabled}
+        onChange={onChange}
+      />
+      {label && (
+        <label htmlFor={label} className={labelClassName}>
+          {label}
+        </label>
+      )}
+      {children}
+    </div>
+  );
+}
+
+RadioOption.defaultProps = {
+  className: '',
+  inputClassName: '',
+  labelClassName: '',
+  label: '',
+  checked: false,
+  disabled: false
```

Comment:
I think none of these `defaultProps` are needed, since this component should behave the same way with `undefs` 🤔